apps/connect/src/hooks/useQueryParams.ts

```diff
@@ -8,12 +9,12 @@ const getChainValue = (
   const sourceChain = query.get(key);
   if (sourceChain) {
     if (isChain(sourceChain)) {
-      return coalesceChainName(sourceChain);
+      return coalesceChainName(sourceChain) as ChainName;
```

Comment:
I'm curious did you check, the types are compatible?, since wormhole connect have started using SDK v2, and it has a different format for chain names